public/app/features/alerting/unified/components/mute-timings/useMuteTimings.tsx

```diff
@@ -192,7 +195,7 @@ export const useGetMuteTiming = ({ alertmanager, name: nameToFind }: BaseAlertma
   useEffect(() => {
     if (useK8sApi) {
       const namespace = getK8sNamespace();
-      getGrafanaTimeInterval({ namespace, name: nameToFind }, true);
+      getGrafanaTimeInterval({ namespace, fieldSelector: `spec.name=${nameToFind}` }, true);
```

Comment:
Does this need to have the field name encoded or quoted? Does this work with any character?

Reply:
Hmm, good point. I just tried naming an interval something silly like

`foo/\/\`

and the API breaks when trying to filter by it with a message of `invalid field selector: invalid escape sequence: \/`

This is happening with the field selector escaped as well:
`fieldSelector=spec.name%3Dfoo%2F%255C%2F%255C`

@yuri-tceretian Are you aware of this limitation on the API?